MoreLinq.sln

```diff
@@ -1,9 +1,10 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26228.4
+VisualStudioVersion = 15.0.26403.7
```

Comment:
This change is unrelated.

Reply:
This was automatically inserted. Unfortunately VS encodes the version in the sln file, so at some point this number should be bumped? Why not now?

Reply:
I'm not going to pick on it but changes in a diff should be related to the commit message and anything else is a distraction. Something like this can be avoided with a `git add -p`.